gokart/parameter.py

```diff
@@ -48,12 +65,28 @@ def default(self, obj):
 
 class ListTaskInstanceParameter(luigi.Parameter):
 
+    def __init__(self, expected_type=None, *args, **kwargs):
```

Comment:
expected_type is for the type of elements in the list.

So, I think expected_elements_type may be a better name.

Reply:
I've changed `expected_type` to `expected_elements_type`.